docs/source/getting_started/index.rst

```diff
@@ -6,7 +6,8 @@ Getting started
 
 The Data Processing Framework (DPF) provides numerical simulation users and engineers with a toolbox
 for accessing and transforming simulation data. DPF can access data from Ansys solver
-result (RST) files as well as from several neutral file formats, including CSV, HDF5, and VTK.
+result files as well as from several neutral `file formats <https://dpf.docs.pyansys.com/version/stable/index.html>`_.
```

Comment:
@MichaelNale instead of linking to stable, which may not be right in the future or depending on many things, you can actually refer to another reference in the doc. We have an example here https://github.com/ansys/pydpf-core/blob/03c6ca54e7d0c03a53c16b6aeb4f795aff1d669f/docs/source/user_guide/plotting.rst?plain=1#L1  with a reference here https://github.com/ansys/pydpf-core/blob/03c6ca54e7d0c03a53c16b6aeb4f795aff1d669f/docs/source/getting_started/install.rst?plain=1#L26

Reply:
```suggestion
result files as well as from several neutral file formats (See :ref:`supported_file_formats`).
```

Reply:
Well the referencing works for section but not for table. So I referenced the main index page